app/controllers/service_providers_controller.rb

```diff
@@ -47,7 +58,16 @@ def show; end
 
   def all
     return unless current_user.admin?
-    @service_providers = ServiceProvider.all.sort_by(&:created_at).reverse
+    prod_apps = ServiceProvider.all.sort_by(&:created_at).reverse.select { 
+                  |sp| sp.prod_config == false 
+                }
+    sandbox_apps = ServiceProvider.all.sort_by(&:created_at).reverse.select { 
+                  |sp| sp.prod_config == false 
+                }
+
+    @service_providers = { 'Production Apps' => prod_apps,
```

Comment:
[suggestion] it feels a little more intuitive to have this be an array of hashes, ie

```ruby
[ 
  {
    type: 'Production Apps',
    apps: prod_apps
  },
  {
    type: 'Sandbox Apps',
    apps: sandbox_apps
  },
]
```

wdyt?

Reply:
I don't hate it, so updating. Wish there was a way to reuse this, but don't want to pass the "My " prefix to a function to build it either. For now, its duplicated for `index` and `all`.

Reply:
hmmm you could do something a little goofy like passing in a local in the erb file? or just get rid of the "my" totally. (but i don't think it's the worst to have it duplicated for now -- we can make a different decision later if necessary)

Reply:
Yeah, it'll live as is for now. Maybe we can convince the new UX person to make a case for removing the "My" then we go from that.